$OpenBSD: patch-src_print_volume_c,v 1.1.1.1 2012/05/02 07:37:10 jasper Exp $

Print volume on OpenBSD and add missing library accordingly (committed upstream).

--- src/print_volume.c.orig	Fri Apr 27 19:04:24 2012
+++ src/print_volume.c	Fri Apr 27 19:04:30 2012
@@ -17,6 +17,12 @@
 #include <sys/soundcard.h>
 #endif
 
+#ifdef __OpenBSD__
+#include <fcntl.h>
+#include <unistd.h>
+#include <soundcard.h>
+#endif
+
 #include "i3status.h"
 #include "queue.h"
 
@@ -166,7 +172,7 @@ void print_volume(yajl_gen json_gen, char *buffer, con
 		}
 	}
 #endif
-#ifdef __FreeBSD__
+#if defined(__FreeBSD__) || defined(__OpenBSD__)
         char mixerpath[] = "/dev/mixer";
         int mixfd, vol, devmask = 0;
 
